contracts/protocol/libraries/logic/ConfiguratorLogic.sol

```diff
@@ -157,7 +162,44 @@ library ConfiguratorLogic {
             encodedCall
```

Comment:
seems we wrap encodedCall as `initialize` and call it again, is that necessary for upgrade or not?